components/Header/index.js

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <>
+      <StyledHeader>
+        <Image src="/logo.png" alt="Logo" width={150} height={125} priority />
+      </StyledHeader>
+    </>
+  );
+}
+
+const StyledHeader = styled.header`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 9.375rem;
```

Comment:
since this element is 'fixed' it is out of document flow. margin-bottom will not have any effect.